routes/index.route.js

```diff
@@ -4,6 +4,13 @@ const router = express.Router();
 const userRoute = require('./user.route');
 const questionRoute = require('./question.route');
 
+// eslint-disable-next-line no-unused-vars
+router.use((err, req, res, _next) => {
+  res.status(500).json({
+    message: 'An error occurred while processing your request',
+  });
+});
+
```

Comment:
Please move it down. It should be the last thing just before `module.exports = router;` 🙏

Reply:
Okay